packages/devtools_app/lib/src/screens/inspector/layout_explorer/ui/utils.dart

```diff
@@ -362,13 +373,15 @@ class AnimatedLayoutProperties<T extends LayoutProperties>
   double get height => size.height;
 
   @override
-  bool get isFlex => begin.isFlex && end.isFlex;
+  bool get isFlex => begin.isFlex == true && end.isFlex == true;
```

Comment:
isFlex is non nullable, so we do not need to use `== true`

Reply:
done